app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -59,16 +68,47 @@ export const ListingDetailsEligibility = ({
     },
   ]
 
+  let occupancySubtitle = ""
```

Comment:
are we capturing all these states with snapshots? I like the idea of that since there's a lot of different ui that can be generated!

Reply:
good point, I can add tests for the separate states 🙂